### ISSUE TITLE
Issues with Gradle 6.0-rc-1

### ISSUE THREAD

**Issue by juergenzimmermann** [jk1/Gradle-License-Report#162]
When I use Version _1.11_ with _Gradle 6.0 rc1_ I get the following output:

```
> Task :generateLicenseReport
Property 'classpath' is declared without normalization specified. Properties of cacheable work must declare their normalization via @PathSensitive, @Classpath or @CompileClasspath. Defaulting to PathSensitivity.ABSOLUTE. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.
Property 'config.filtersCache' is private and annotated with @Nested. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.
Property 'config.importersCache' is private and annotated with @Nested. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.
Property 'config.licenseReportExtensionSnapshot' is private and annotated with @Input. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.
Property 'config.renderersCache' is private and annotated with @Nested. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.
Property 'config.renderer' is not annotated with an input or output annotation. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.
Property 'config.renderersCache.$0.fileNameCache' is private and annotated with @Input. This behaviour has been deprecated and is scheduled to be removed in Gradle 7.0.
The runtime configuration has been deprecated for resolution. This will fail with an error in Gradle 7.0. Please resolve the runtimeClasspath configuration instead.
```

See also https://github.com/jk1/Gradle-License-Report/issues/161.

**Comment by jk1** [comment]
Thank you for reporting it, this is indeed to be fixed.
The last warning can  be resolved by means of configuration though, since the plugin allows a build author to specify the configuration to collect dependencies from.

**Comment by juergenzimmermann** [comment]
@jk1 Thank you for the hint. Using `licenseReport { configurations = arrayOf("runtimeClasspath") }` the last warning is gone.

**Comment by jk1** [comment]
Fixed in #159
